ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,7 +56,7 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
+        //this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
I'm all for push notifications. But it is out of scope of this PR and it might require some more changes. We keep it as it is for now and get back to it later.